User-defined API load testing benchmarks [RN35880] [DCP206612] (#1205)
SkylineCommunications/dataminer-docs: user-guide/Reference/Metrics/user-defined_API_benchmarks.md

---
uid: user-defined_API_benchmarks
---

# User-defined API benchmarks

> ## Specifications of the test VM server 1

Reviewer: Not sure why a "1" is added if only one server is mentioned. I'd suggest to keep it simple:

```suggestion
## Specifications of the test VM server
```

- Intel Core i7-12700
- 8 cores
- 32GB RAM
- HDD
- Windows 10

> ## Specifications of the test client PC 1

Reviewer: ```suggestion
## Specifications of the test client
```

Author: Done

- Intel Core i7-12700
- 12 cores
- 32GB DDR-5 RAM
- SSD
- Windows 11

## Benchmarks

| \# | Specification | Scope | Metric | Configuration |
| -- | ------------- | ----- | ------ | ------------- |

> | 1 | Server call duration of 2500 API tokens and 2500 API definitions from server 1 to client 1 | DMA | 2s - 2.5s | Clean DMA, no other data |

Reviewer: The current phrasing is a bit unclear. Is this a correct interpretation?

```suggestion
| 1 | Time for the server to retrieve 2500 API tokens and 2500 API definitions | DMA | 2s - 2.5s | Clean DMA without other data |
```

> | 2 | Load testing of 2500 API tokens and 2500 API definitions in DataMiner Cube of client 1 | Cube | 3s - 3.5s | Clean DMA, no other data |

Reviewer: ```suggestion
| 2 | Time for the client to load 2500 API tokens and 2500 API definitions in DataMiner Cube | Cube | 3s - 3.5s | Clean DMA without other data |
```

Author: Done
